plugins.json

```diff
@@ -83,6 +83,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.32"
     },
+    "bigboard": {
+        "title": "Bigboard",
+        "version": "1.0.2",
+        "author": "Stinnux",
+        "license": "MIT",
+        "description": "A Kanboard that can display multiple projects.",
+        "homepage": "https://github.com/stinnux/kanboard-bigboard",
+        "readme": "https://raw.githubusercontent.com/stinnux/kanboard-bigboard/master/README.md",
+        "download": "https://github.com/stinnux/kanboard-bigboard/releases/download/1.0.2/Bigboard-1.0.2.zip",
+        "remote_install": true,
+        "compatible_version": ">=1.2.22"
```

Comment:
1.2.22 doesn't exists yet. The latest version is 1.2.3, see https://github.com/kanboard/kanboard/releases